components/input/ColorPicker.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-components';
+import { ChromePicker } from 'react-color';
+
+const ColorPicker = ({ text, initialColor, onColorChange, ...rest }) => {
+    const [pickerState, setPickerState] = useState({
+        display: false,
+        color: initialColor
+    });
+
+    const rgbaColor = (color) => {
+        return `rgba(${color.rgb.r}, ${color.rgb.g}, ${color.rgb.b}, ${color.rgb.a})`;
+    };
+
+    const handleClick = () => {
+        setPickerState((prevState) => {
+            return {
+                ...prevState,
+                display: !pickerState.display
+            };
+        });
+    };
+
+    const handleClose = () => {
+        setPickerState((prevState) => {
```

Comment:
You can use a shorter version also here.

Reply:
See comment above